pkg/zfs/plugin/zfs.go

```diff
@@ -37,7 +39,7 @@ const (
 	ZfsPvNamespace = "namespace"
 
 	// ZfsPvBackup config key for backup type full or incremental
-	ZfsPvBackup = "backup"
+	ZfsPvIncr = "incr"
```

Comment:
nit: `incr` seems less descriptive. Since this is user facing parameter, I'll say better to have more descriptive name.